r/R/examples/stat_large.R

```diff
@@ -0,0 +1,36 @@
+#source("../core.R")
+#source("../ui.R")
```

Comment:
We can probably remove these imports from all the examples.

Reply:
removed from examples